Index.html

```diff
@@ -20,358 +20,663 @@
     <link rel="stylesheet" href="style.css" />
   </head>
 
-  <body>
-    <header class="header-font">
-      <nav>
-        <a href="#">
-          <!-- <img src="" alt="Binabox" /> -->
-        </a>
-        <ul class="navigation">
-          <li><a href="#">Home</a></li>
-          <li><a href="#">Blog</a></li>
-          <li><a href="#">Catalog</a></li>
-          <li><a href="#">Contact Us</a></li>
+  <body class="body__homepage">
+    <header class="site-header">
+      <a href="#" class="site-header__logo">
+        <img src="#" alt="Binabox" class="site-header__logo-image" />
+      </a>
+
+      <nav class="site-header__menu">
+        <ul class="site-header__list">
+          <li class="site-header__item">
+            <a
+              class="site-header__link site-header__link--active"
+              href="index.html"
+              >Home</a
+            >
+          </li>
+          <li class="site-header__item">
+            <a class="site-header__link" href="blog.html">Blog</a>
+          </li>
+          <li class="site-header__item">
+            <a class="site-header__link" href="#">Catalog</a>
+          </li>
+          <li class="site-header__item">
+            <a class="site-header__link" href="#">Contact Us</a>
+          </li>
         </ul>
-        <div class="nav-actions">
-          <a href="https://discord.com" target="_blank">Discord</a>
-          <a href="/login.html">Log In</a>
-        </div>
       </nav>
+
+      <div class="site-header__utilities">
```

Comment:
"This class seems unnecessarily long. I would suggest renaming it to header__actions, which makes it easier to understand its purpose."